rpm_spec/manageiq.spec.in

```diff
@@ -47,18 +47,12 @@ pathfix.py -pni "%{__python3} %{py3_shbang_opts}" %{gemset_builddir}
 %build
 cd %{_builddir}
 
-cat <<"EOF" > %{gemset_builddir}/enable
-export APPLIANCE="true"
-export BUNDLE_GEMFILE=%{app_root}/Gemfile
-export GEM_HOME=%{gemset_root}
-export GEM_PATH=%{gemset_root}:$(gem env path)
-export PATH=%{gemset_root}/bin:$PATH
-EOF
-
-cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/evm_production
-export APPLIANCE_SOURCE_DIRECTORY=%{appliance_root}
-export APPLIANCE_TEMPLATE_DIRECTORY=${APPLIANCE_SOURCE_DIRECTORY}/TEMPLATE
-source %{gemset_root}/enable
+cat <<"EOF" > %{appliance_builddir}/LINK/etc/default/production.properties
+APPLIANCE_SOURCE_DIRECTORY=%{appliance_root}
+APPLIANCE_TEMPLATE_DIRECTORY=%{appliance_root}/TEMPLATE
+GEM_HOME=%{gemset_root}
+GEM_PATH=%{gemset_root}:/usr/share/gems:/usr/local/share/gems
+PATH=%{gemset_root}/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
```

Comment:
Same with this one - Just want to verify that we don't clobber something in either env.